lib/logmsg/tests/test_logmsg_serialize.c

```diff
@@ -239,12 +239,11 @@ Test(logmsg_serialize, simple_serialization)
 
   log_msg_deserialize(msg, sa);
 
-  LogStamp ls =
-  {
-    .tv_sec = msg->timestamps[LM_TS_RECVD].tv_sec,
-    .tv_usec = msg->timestamps[LM_TS_RECVD].tv_usec,
-    .zone_offset = msg->timestamps[LM_TS_RECVD].zone_offset
-  };
+  LogStamp ls;
```

Comment:
Why did you drop designated initializers? (I think that initialization way is superior).

Reply:
I agree, the reason that the UnixStamp was a member of LogStamp, but at the end I've decided against it, and alias LogStamp to UnixTime, with that the initializers could come back.

So agreed, thanks for catching this.

Reply:
I've sent a followup patch to revert this piece and go back to C99 initializers.